app/services/data/get-app-workloads.js

```diff
@@ -5,9 +5,7 @@
   const Workload = require('wmt-probation-rules').Workload
   const Locations = require('wmt-probation-rules').Locations
 
-  module.exports = function (initialId, batchSize) {
```

Comment:
Any kind of constraint here ensuring that batchSize is > 0?

Reply:
Nothing, but we can do a check and throw an error if it isn't.